server/conf/application.conf

```diff
@@ -14,6 +14,13 @@ play.crypto.secret = "changeme"
 # ~~~~~
 play.i18n.langs = [ "en" ]
 
+jira.baseUrl = "https://jira.agiledigital.com.au"
```

Comment:
We need ticket endPoint. Cause this is used for fetching ticket information only. Later on we need to append other endPoints to construct different URL based on JIRA base URL for other purposes.